parse_test.go

```diff
@@ -347,6 +347,8 @@ func filterData(data ParserData) ParserData {
 		"md5_digest",
 		"sha256_digest",
 		"version",
+		"description",
+		"author",
```

Comment:
Do we know why these changed or what these changed to? It feels weird that `author` changed, but not `author_email`. It's starting to feel more like we should store files in this repo so we aren't cloning the latest. That way we don't have to ignore any fields and if anything changes, it's because `twine` behavior changed.

Reply:
The author field was set to `<name> + (n others)`. As a project gains contributors, `n` increases, so the snapshot doesn't match.

> It's starting to feel more like we should store files in this repo so we aren't cloning the latest.

I'm not completely against it, but I don't feel like it's worth spending time on this right now, especially considering how infrequently Twine releases, and how the upload code has been fairly stable: https://pypi.org/project/twine/#history

Reply:
Makes sense! We can leave this as-is for now, but as more fields change and we set more to `<field name> exists`, these tests lose more of their usefulness in my opinion. Storing the distributions with the tests would reduce the variability to only catching the random, infrequent Twine change, which is the benefit of these tests to me.

Reply:
I agree with you. I'll pin the repo references.

Reply:
Great idea, thanks for doing that!